src/OnboardingSPA/components/OptionWithHeadingSubHeading/stylesheet.scss

```diff
@@ -9,14 +9,28 @@ $background-color: var(--nfd-onboarding-card-background);
 		align-items: center;
 		border-radius: 8px;
 		padding: 16px 12px 16px 12px;
-		justify-content: space-between;
 		transition: background-color 400ms ease-in-out;
 
 		&--selected {
 			background-color: var(--nfd-onboarding-hover);
 		}
 	}
 
+	&__radio_button {
+		margin-left: 5px;
+		margin-right: 15px;
+
+		input[type="radio"] {
```

Comment:
Should we be targetting this or can we assign a class and target that? or check name? `name="nfd__option_radio_button"` if that is possible?